sonic_y_cable/y_cable_base.py

```diff
@@ -803,6 +880,11 @@ def restart_anlt(self):
         The port on which this API is called for can be referred using self.port.
 
         Args:
+            target:
+                One of the following predefined constants, the actual target to restart anlt on:
```

Comment:
Replace "anlt" with "AN/LT". Do this throughout all docstrings for consistency.

Reply:
fixed